nohup: ignoring input
2020-03-23 20:45:00.265790: W tensorflow/stream_executor/platform/default/dso_loader.cc:55] Could not load dynamic library 'libnvinfer.so.6'; dlerror: libnvinfer.so.6: cannot open shared object file: No such file or directory
2020-03-23 20:45:00.265992: W tensorflow/stream_executor/platform/default/dso_loader.cc:55] Could not load dynamic library 'libnvinfer_plugin.so.6'; dlerror: libnvinfer_plugin.so.6: cannot open shared object file: No such file or directory
2020-03-23 20:45:00.266012: W tensorflow/compiler/tf2tensorrt/utils/py_utils.cc:30] Cannot dlopen some TensorRT libraries. If you would like to use Nvidia GPU with TensorRT, please make 27.17.246.179 - - [23/Mar/2020 20:45:05] "[37mGET / HTTP/1.1[0m" 200 -
27.17.246.179 - - [23/Mar/2020 20:45:07] "[37mGET / HTTP/1.1[0m" 200 -
oader.cc:55] Could not load dynamic library 'libcuda.so.1'; dlerror: libcuda.so.1: cannot open shared object file: No such file or directory
2020-03-23 20:45:01.384295: E tensorflow/stream_executor/cuda/cuda_driver.cc:351] failed call to cuInit: UNKNOWN ERROR (303)
2020-03-23 20:45:01.384325: I tensorflow/stream_executor/cuda/cuda_diagnostics.cc:156] kernel driver does not appear to be running on this host (iZj6cik2sm4xzs7rhx2o7jZ): /proc/driver/nvidia/version does not exist
2020-03-23 20:45:01.384563: I tensorflow/core/platform/cpu_feature_guard.cc:142] Your CPU supports instructions that this TensorFlow binary was not compiled to use: AVX2 FMA
2020-03-23 20:45:01.397169: I tensorflow/core/platform/profile_utils/cpu_utils.cc:94] CPU Frequency: 2499995000 Hz
2020-03-23 20:45:01.397445: I tensorflow/compiler/xla/service/service.cc:168] XLA service 0x558031320da0 initialized for platform Host (this does not guarantee that XLA will be used). Devices:
2020-03-23 20:45:01.397462: I tensorflow/compiler/xla/service/service.cc:176]   StreamExecutor device (0): Host, Default Version
  Suel both to cruet with makest is me thee,
When that all the very commor doth did,
The beauty leasure then my age in ever,
Giring that beauty shall the world and love.
  Then be for my winter's fair and curncting

That every wand when thou art all the friend,
Which in my poor ride of love as in
Hor having compore to his sweet beauty's bare
Which shall alove that which it hath sweet
With up true shall still with me as onte.

That is thy groud saive health in my pout;
On that a cappe to me art a
 * Serving Flask app "views" (lazy loading)
 * Environment: production
   WARNING: This is a development server. Do not use it in a production deployment.
   Use a production WSGI server instead.
 * Debug mode: on
Using TensorFlow backend.
Traceback (most recent call last):
  File "views.py", line 15, in <module>
    app.run(debug = True,host='0.0.0.0',port=5000)
  File "/root/anaconda3/lib/python3.7/site-packages/flask/app.py", line 990, in run
    run_simple(host, port, self, **options)
  File "/root/anaconda3/lib/python3.7/site-packages/werkzeug/serving.py", line 1030, in run_simple
    s.bind(server_address)
OSError: [Errno 98] Address already in use
